packages/front-end/services/UserContext.tsx

```diff
@@ -462,6 +430,37 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     if (data) setReady(true);
   }, [data]);
 
+  const organization: Partial<OrganizationInterface> | undefined =
+    currentOrg?.organization;
+  const subscription = currentOrg?.subscription || null;
```

Comment:
@jdorn Is this going to break for orgs with Stripe subscriptions that started before we introduced the license server? 

It looks like the subscription can either live at `currentOrg.subscription`, while we also have metadata that lives at `currentOrg.organization.subscription`